src/sentry/monitors/clock_dispatch.py

```diff
@@ -43,9 +60,15 @@ def _dispatch_tick(ts: datetime):
     skip any tasks it missed)
     """
     if options.get("crons.use_clock_pulse_consumer"):
-        # TODO(epurkhiser): This should dispatch the pulse as a message on the
-        # monitors-clock-pulse topic
-        pass
+        if settings.SENTRY_EVENTSTREAM != "sentry.eventstream.kafka.KafkaEventStream":
+            # XXX(epurkhiser): Unclear what we want to do if we're not using kafka
```

Comment:
Just fire the tasks directly probably, it's only for dev

Reply:
Yeah I'll keep the logic separate to handle that 👍